src/ServiceControl.RavenDB/StartupChecks.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace ServiceControl.RavenDB
+{
+    using System.Reflection;
+    using System.Threading;
+    using Raven.Client.Documents;
+    using Raven.Client.Documents.Conventions;
+    using Raven.Client.Http;
+    using Raven.Client.ServerWide.Operations;
+    using Sparrow.Json;
+
+    public static class StartupChecks
+    {
+        public static async Task EnsureServerVersion(IDocumentStore store, CancellationToken cancellationToken = default)
+        {
+            var build = await store.Maintenance.Server.SendAsync(new GetBuildNumberOperation(), cancellationToken);
+            var embeddedVersion = typeof(Raven.Embedded.EmbeddedServer).Assembly.GetCustomAttributes<AssemblyInformationalVersionAttribute>().First().InformationalVersion;
+            if (embeddedVersion != build.FullVersion)
+            {
+                throw new Exception($"ServiceControl expects RavenDB Server version {embeddedVersion} but the server is using {build.FullVersion}.");
+            }
+        }
+
+        public static async Task EnsureServiceControlLicense(IDocumentStore store, CancellationToken cancellationToken = default)
+        {
+            var licenseResponse = await store.Maintenance.Server.SendAsync(new GetLicenseOperation(), cancellationToken);
+            if (licenseResponse.LicensedTo != "ParticularNservicebus (Israel)")
+            {
+                throw new Exception("Wrong license");
+            }
+        }
```

Comment:
Do we need this anymore since we are rolling our own image?

The important check is the version. If they want the license they use our image. I think we can remove this check.